gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/SpecimenMapperTest.java

```diff
@@ -162,6 +240,36 @@ private static DiagnosticReport generateDiagnosticReportWithNoSpecimenReference(
         return (DiagnosticReport) new DiagnosticReport().setId(DR_ID);
     }
 
+    private void assertMetaSecurityIsPresent(final Meta meta) {
+        final List<Coding> metaSecurity = meta.getSecurity();
+        final int metaSecuritySize = metaSecurity.size();
+        final Coding metaSecurityCoding = metaSecurity.getFirst();
+        final UriType metaProfile = meta.getProfile().getFirst();
+
+        assertAll(
+            () -> assertThat(metaSecuritySize).isEqualTo(1),
+            () -> assertThat(metaProfile.getValue()).isEqualTo(SPECIMEN_META_PROFILE),
+            () -> assertThat(metaSecurityCoding.getCode()).isEqualTo(NOPAT_CV.getCode()),
+            () -> assertThat(metaSecurityCoding.getDisplay()).isEqualTo(NOPAT_CV.getDisplayName()),
+            () -> assertThat(metaSecurityCoding.getSystem()).isEqualTo(NOPAT_CV.getCodeSystem())
+        );
+    }
```

Comment:
Just thinking, assuming that the parameter `meta` is the resulting meta of a test, wouldn't it make more sense to use `assertThat(meta).usingRecursiveComparison().isEqualTo(META_WITH_SECURITY_ADDED);`?